app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -121,24 +120,30 @@ module.exports = React.createClass
   handleFocus: ->
     @menus[0].focus()
 
-  getOptions: (i, optionsKeys) ->
-    optionsKey = if optionsKeys.length then optionsKeys.join(';') else '*'
+  getOptionsKey: (i, optionsKey = '') ->
+    {selects} = @props.task
+    select = selects[i]
+    [condition] = selects.filter (filterSelect) => filterSelect.id is select.condition
```

Comment:
This can probably be speeded up so that it doesn't have to loop through the `selects` array every time it's called.

Maybe we should store a lookup table of menu ID => selected value.